lua/neo-tree/ui/renderer.lua

```diff
@@ -510,7 +520,7 @@ end
 ---Draws the given nodes on the screen.
 --@param nodes table The nodes to draw.
 --@param state table The current state of the source.
-local draw = function(nodes, state, parent_id)
+draw = function(nodes, state, parent_id)
```

Comment:
Same here, without the local, isn't it global?

Reply:
Same as previous; it is declared local at the top of the file (I assume for a forward declaration, but did not look further into it).